omod/src/main/java/org/openmrs/web/servlet/LoginServlet.java

```diff
@@ -65,6 +65,8 @@
 	 * @see javax.servlet.http.HttpServlet#doPost(javax.servlet.http.HttpServletRequest,
 	 *      javax.servlet.http.HttpServletResponse)
 	 */
+	private Integer loginAttemptsByUser;
```

Comment:
Can you explain a bit why you made this a class level instead of local variable?

Reply:
oops , soory. i actually had seen as thought am puting the code in a different method from the "doPost" method. let me correct that